privacyidea/static/components/dashboard/controllers/dashboardControllers.js

```diff
@@ -149,24 +163,40 @@ myApp.controller("dashboardController", function (ConfigFactory, TokenFactory,
         return 0;
      };
 
-    $scope.get_total_token_number();
-    $scope.get_token_hardware();
-    $scope.get_token_software();
-    $scope.get_policies();
-    $scope.get_events();
-    $scope.getSubscriptions();
-    $scope.getAuthentication();
-    $scope.getAdministration();
-
-        // listen to the reload broadcast
-    $scope.$on("piReload", function() {
+    if (AuthFactory.checkRight('tokenlist')) {
```

Comment:
If you are doing the check here, you could actually omit it in the functions get_total_token_number, get_token_hardware and get_token_software.

Reply:
correct. Done in https://github.com/privacyidea/privacyidea/pull/2469/commits/c69eade84c8530f5f04e982a89907e4eac0a9fbf